Validation/CaloTowers/test/run_onRelVal_data_cfg.py

```diff
@@ -0,0 +1,152 @@
+import os
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("hcalval")
+process.load("Configuration.StandardSequences.Reconstruction_cff")
+process.load("Configuration.StandardSequences.Geometry_cff")
```

Comment:
This is the wrong geometry file to load. Please replace it with GeometryRecoDB_cff instead.